src/taxpasta/infrastructure/application/standard_profile_writer/ods_standard_profile_writer.py

```diff
@@ -35,7 +33,6 @@ def write(
         cls,
         profile: DataFrame[StandardProfile],
         target: BinaryBufferOrFilepath,
-        taxonomy: Optional[Taxonomy] = None,
```

Comment:
More of a question but why is this argument removed?

Reply:
Taxonomy information is now simply added to the table before writing it out. So the `profile` argument is in fact no longer a simple `StandardProfile` but may contain `name`, `rank`, and `lineage` columns.